src/SFML/Graphics/Color.cpp

```diff
@@ -68,6 +68,24 @@ a(alpha)
 
 
 ////////////////////////////////////////////////////////////
+Color::Color(Uint32 color) :
+r((color & 0xff000000) >> 24),
+g((color & 0x00ff0000) >> 16),
+b((color & 0x0000ff00) >> 8 ),
+a((color & 0x000000ff) >> 0 )
+{
+
+}
+
+
+////////////////////////////////////////////////////////////
+Uint32 Color::toInteger()
+{
+    return r << 24 | g << 16 | b << 8 | a;    
```

Comment:
Just tried it, there's an implicit conversion happening.

    unsigned char a = 5;
    std::cout << (a << 8) << std::endl;

Result is indeed 1280.